curricula/models.py

```diff
@@ -454,46 +458,31 @@ def jackfrost_urls(self):
     def pdf_urls(self):
         return [self.get_pdf_url(), self.get_resources_pdf_url()]
 
-    def jackfrost_can_build(self):
```

Comment:
This doesn't seem safe to delete, isn't it called in other places? (Like `unit_handler` at the bottom of this file?)

Reply:
good catch! I thought both functions were in the same class. Will bring back the old function. It'd be good to add unit tests too.